lib/Common/Memory/RecyclerPointers.h

```diff
@@ -227,7 +227,7 @@ template <typename T>
 class NoWriteBarrierField
 {
 public:
-    NoWriteBarrierField() {}
+    NoWriteBarrierField() : value() {}
```

Comment:
That makes it "value initialized". Otherwise value may not be initialized, e.g. if value is of type int.